docs/source/overview/index.rst

```diff
@@ -0,0 +1,53 @@
+
+3. ``main.py`` reads a *time activity multiplier* which is a file that is used to change peoples' normal behaviour so that them spend more time at home (e.g. during a lockdown).
+
+4. Finally, ``main.py`` determines whether the OpenCL model or the Python model should be executed and passes the required arguments to the appropriate classes. 
+
+The Python and OpenCL models are introduced in more detail below, but broadly one of two things will happen:
+
+* If the **OpenCL** model is executed: a secondary cache, called a *snapshot* is created or read in if the model has been executed once already. Basically the model takes the pandas DataFrames created earlier and converts them into arrays. The ``microsim/opencl/ramp/run.py`` file is called and a ``Simulator`` class (``microsim/opencl/ramp/simualtor.py``) is created and stepped to run the model.
+
+* If the **Python** model is executed: a ``Microsim`` object, see  :ref:`microsim/microsim_model.py<microsim-label>`, is created to run the model by calling ``Microsim.run()`` Note that if the model is going to be run multiple times with the same parameters (*repetitions*) then a multi-process wrapper calls a few ``Microsim.run()`` functions simultaneously using different processes.
+
+
+.. _population_initialisation-label:
+
+Population Initialisation (``microsim/population_initialisation.py``)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The task of the ``PopulationInitialisation`` class is to read the data that represents the synthetic population at the core of the model. This includes csv files with information about each individual, as well as files used to estimate workplace locations, etc. That file is commented quite extensively, so see the documentation there for further details.
+
+The most important thing that the ``PopulationInitialisation`` class does is to return the ``individuals`` dataframe and the ``activity_locations`` dictionary (that points to dataframes for the locations where indvidiauals will spend their time).
+
+
+.. _microsim-label:
+
+Microsim class (``microsim/microsim_model.py``)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``Microsim`` class is the main entry point for the python file. It contains the main code to control the spread of the disease. See the ``step()`` function for details about what, specifically, happens in each iteration of the model. The OpenCL model follows the same procedures, although they are implemented differently.
```

Comment:
`python file` to `python model` - I think just to maintain consistency of how we refer to it in the rest of the docs

Reply:
Done